README.md

```diff
@@ -43,6 +43,12 @@ import('n2words').then(({default: n2words}) => {
 })
 ```
 
+Or:
+```js
+const { default: n2words } = await import("n2words");
```

Comment:
Unfortunately, CommonJS does not have top-level await - https://github.com/nodejs/node/issues/21267

Reply:
The above line work in node console. Should we put it under ESM section ? @TylerVigario

Reply:
@forzagreen This would work in an ESM environment absolutely. Basically, the issue is that prior to the ESM ideology, everything was synchronous. CJS (now) supports async by the use of certain functions, but top-level will never be backward compatible.